api-reference/50 Common/Object Structures/positionConfig/collision/collision.md

```diff
@@ -5,48 +5,127 @@ type: String | Object
 ---
 ---
 ##### shortDescription
-Specifies how to move the widget if it overflows the screen.
+Specifies how to resolve collisions between the overlay and the [boundary](/Documentation/ApiReference/Common/Object_Structures/positionConfig/#boundary) elements.
 
 ---
-A string passed to this option should contain a collision handler name, or a pair of names separated by space. If the string consists of a single collision handler name, it is applied to both horizontal and vertical axes. If you pass a pair of names separated by space, the first collision handler will be applied to the horizontal axis, the second to the vertical axis.
+Collisions occur whenever the specified position causes the overlay element to exceed the boundaries.
 
-    <!--JavaScript-->
-    // a string containing horizontal and vertical
-    // collision handlers separated by space
-    collision: "flip fit";
+You can use the following collision resolution algorithms:
 
-You can also specify different collision handlers for horizontal and vertical axes in one of the following ways.
+- *"flip"*      
+Move the overlay element to the opposite side of the [target](/Documentation/ApiReference/Common/Object_Structures/positionConfig/#of) if there is more available space there.
 
-    <!--JavaScript-->
-    // an object containing 'x' and 'y' fields
-    collision: {
-        x: "flip", // horizontal collision handler
-        y: "none" // vertical collision handler
+- *"fit"*       
+Move the overlay element to the inside of the [boundary](/Documentation/ApiReference/Common/Object_Structures/positionConfig/#boundary) element.
+
+- *"flipfit"*       
+Apply first *"flip"*, then *"fit"*
+
+- *"none"*      
+Ignore the collision.
+
+To set the **collision** option, use an object with the **x** and **y** fields. **x** specifies how to resolve collisions that occur in the horizontal direction, and **y** specifies the same in the vertical direction. Alternatively, you can use one of the string shortcuts listed in the accepted values.
```

Comment:
Can you be more specific than "use an object with the **x** and **y** fields"
(See my previous similar comments above)

Also, for the second sentence can we say:
**x** specifies how to resolve horizontal collisions and **y** specifies vertical collisions.

For the third sentence:
Alternatively, you can use a string shortcut from the accepted values list.

Reply:
Okay